### ISSUE TITLE
code example from the 2.2.0 documentation fails

### ISSUE THREAD

**Issue by fabrice-ducos** [SciTools/iris#3339]
In the "Loading Iris cubes" section, the following example code is provided.

```
import iris
filename = iris.sample_data_path('uk_hires.pp')
cubes = iris.load(filename)
print(cubes)
```

When attempting to test this code (configuration: OSX Mojave, python 3.7.3, anaconda 3, iris 2.2.0, sample data properly installed), I get the following error on the _iris.load()_ call:

```
    721 
    722     # Ensure we have write permission on the data buffer.
--> 723     data.setflags(write=True)
    724 
    725     # Ensure the data is in the native byte order

ValueError: cannot set WRITEABLE flag to True of this array
```

**Comment by pp-mo** [comment]
This broke at numpy 1.16, has been fixed in 2.2.1 : #3257

**Comment by bjlittle** [comment]
Thanks for raising this @fabrice-ducos 

As @pp-mo mentions, this has been fixed in our latest release of `iris` version `2.2.1`.

I'm going to close this issue as it has already been resolved. Please install the latest version of `iris` which is available from `conda-forge`. Thanks!

```python
>>> import iris
>>> iris.__version__
'2.2.1dev0'
>>> import numpy as np
>>> np.__version__
'1.16.4'
>>> fname = iris.sample_data_path('uk_hires.pp')
>>> cubes = iris.load(fname)
>>> print(cubes)
0: air_potential_temperature / (K)     (time: 3; model_level_number: 7; grid_latitude: 204; grid_longitude: 187)
1: surface_altitude / (m)              (grid_latitude: 204; grid_longitude: 187)
```
